Temporary patch until accel-pptp wil be stable inside fw

diff -BurpN gateway/rc.orig/network_ex.c gateway/rc/network_ex.c
--- gateway/rc.orig/network_ex.c	2008-03-19 17:07:24.000000000 +0000
+++ gateway/rc/network_ex.c	2009-08-17 14:45:09.000000000 +0000
@@ -128,6 +128,18 @@ int start_pppd(char *prefix)
 	fprintf(fp, "password '%s'\n", 
 		nvram_safe_get(strcat_r(prefix, "pppoe_passwd", tmp)));
        	       	
+	if (nvram_match(strcat_r(prefix, "proto", tmp), "pptp") &&
+	    nvram_match(strcat_r(prefix, "pptp_kernel", tmp), "1"))
+	{
+		fprintf(fp, "plugin pptp.so\n");
+		fprintf(fp, "pptp_server %s\n", 
+   			nvram_invmatch("wan_heartbeat_x", "") ?
+   			nvram_safe_get("wan_heartbeat_x") : 
+   			nvram_safe_get(strcat_r(prefix, "pppoe_gateway", tmp))); 
+		/* see KB Q189595 -- historyless & mtu */
+		fprintf(fp, "nomppe-stateful %s mtu 1400\n",
+			nvram_safe_get(strcat_r(prefix, "pptp_options_x", tmp)));
+ 	} else
        	if (nvram_match(strcat_r(prefix, "proto", tmp), "pptp")) 
        	{
 		fprintf(fp, "connect true\n");    
